lib/internal/Magento/Framework/Escaper.php

```diff
@@ -476,4 +485,32 @@ private function getTranslateInline()
 
         return $this->translateInline;
     }
+
+    /**
+     * Inline sensitive escape attribute value.
+     *
+     * @param string $text
+     * @return string
+     */
+    private function inlineSensitiveEscapeHthmlAttr(string $text): string
+    {
+        $escaper = $this->getEscaper();
```

Comment:
Please check that string has length more or eq 6 before.

Reply:
Thank you, fixed.